lib/exporters/jhipster_entity_exporter.js

```diff
@@ -20,7 +20,7 @@
 const fs = require('fs');
 const path = require('path');
 const ApplicationTypes = require('../core/jhipster/application_types');
-const { toFilePath, readJSONFile } = require('../readers/json_file_reader');
+const { toFilePath, toPreviousStateFilePath, readJSONFile } = require('../readers/json_file_reader');
```

Comment:
I don't get what the function does just by looking at the name.
What does it do?

Reply:
I've suggested another name, perhaps it's more suggestive